src/adagrams.js

```diff
@@ -1,8 +1,162 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    let letters = ['a', 'a', 'a', 'a', 'a', 'a', 'a', 'a', 'a', 'b', 'b', 'c', 'c', 'd', 'd', 'd', 'd', 'e', 'e', 'e', 'e', 'e', 'e', 'e', 'e', 'e', 'e', 'e', 'e', 'f', 'f', 'g', 'g', 'g', 'h', 'h', 'i', 'i', 'i', 'i', 'i', 'i', 'i', 'i', 'i', 'j', 'k', 'l', 'l', 'l', 'l', 'm', 'm', 'n', 'n', 'n','n', 'n', 'n', 'o', 'o', 'o', 'o', 'o', 'o', 'o', 'o', 'p', 'p', 'q', 'r', 'r', 'r', 'r', 'r', 'r', 's', 's', 's', 's', 't', 't', 't', 't', 't', 't', 'u', 'u', 'u', 'u', 'v', 'v', 'w', 'w', 'x', 'y', 'y', 'z'];
+
+    const lettersInHand = [];
+    // Method from Stack Overflow here: https://stackoverflow.com/questions/5915096/get-random-item-from-javascript-array
+    // Math.random returns a floating point between 0-1, multiplies that by the length of the array, and Math.floor rounds that down to determine a 'random' index - is it really random?  According to some people, no.  But we'll take it for our first go at a randomizer. 
+    for(let i = 0; i < 10; i++) {
+      let randomIndex = Math.floor(Math.random()*letters.length);
+      let letter = letters[randomIndex];
+      
+      letters.splice(randomIndex, 1);
+
+      lettersInHand.push(letter.toUpperCase());
+    };
+
+    return lettersInHand;
+  },
+
+  usesAvailableLetters(input, lettersInHand) {
+    const lettersInHandCopy = lettersInHand.map((letter) => letter);
+  
+    const splitInput = input.split('');
+        
+    for (let i = 0; i < splitInput.length; i++) {
+      if (lettersInHandCopy.includes(splitInput[i])) {
+        const index = lettersInHandCopy.indexOf(splitInput[i]);
+        lettersInHandCopy.splice(index, 1);
+      } else {
+        return false;
+      };
+  
+    };
+    return true;
+  },
+
+  scoreWord(word) {
+    let score = 0;
+  
+    if (word.length > 6) {
+      score += 8;
+    };
+  
+    const wordArray = word.toUpperCase().split('');
+  
+    for (const char of wordArray) {
+      switch(char) {
+        case 'A':
+        case 'E': 
+        case 'I': 
+        case 'O': 
+        case 'U': 
+        case 'L': 
+        case 'N': 
+        case 'R': 
+        case 'S':
+        case 'T':
```

Comment:
nice ues of a switch case and its ability to have cases "fall" to the same behavior!